src/test/java/teammates/test/cases/browsertests/FeedbackMcqQuestionUiTest.java

```diff
@@ -59,6 +59,23 @@ public void testNewQuestionFrame() {
         feedbackEditPage.clickNewQuestionButton();
         feedbackEditPage.selectNewQuestionType("MCQ");
         assertTrue(feedbackEditPage.verifyNewMcqQuestionFormIsDisplayed());
+
+        feedbackEditPage.clickGenerateMcqOptionsCheckbox(-1);
```

Comment:
I believe the test case should start here?
`______TS("MSQ: Check UI after cancelling and add new MCQ question again");`